html_app/ui/facs.soy

```diff
@@ -301,31 +305,37 @@ FACS Main View
         </thead>
         {foreach $r in $rows}
         <tr class='scb_s_facs_samples_table_tr {if $r.display_sample} scb_s_microscopy_row_border{/if}'  role='row' >
-            <td class='scb_s_facs_samples_table_td {if $r.display_sample}{else}scb_s_experiment_setup_table_border{/if} ' style='width:56px;'>{if $r.display_sample}
-                <input  type="checkbox" class="scb_f_facs_sample_active" facs_id='{$facs.id}'  aria-checked='{if $r.is_sample_enabled}true{else}false{/if}'
-                       assignment_id='{$assignment.id}' experiment_id='{$experiment.id}'
-                       cell_treatment_id='{$r.cell_treatment.id}'  role='checkbox'
-                {if $r.is_sample_enabled}checked="checked"{/if}
-                >
+            <td class='scb_s_facs_samples_table_td {if $r.display_sample}{else}scb_s_experiment_setup_table_border{/if} '>
+                {if $r.display_sample}
+                    <input  type="checkbox" class="scb_f_facs_sample_active" facs_id='{$facs.id}'
+                        aria-checked='{if $r.is_sample_enabled}true{else}false{/if}'
+                        assignment_id='{$assignment.id}' experiment_id='{$experiment.id}'
+                        cell_treatment_id='{$r.cell_treatment.id}'  role='checkbox'
+                        {if $r.is_sample_enabled}checked="checked"{/if}
+                    >
                 {/if}
-            </td> {if $r.kind == 'existing' and length(keys($kinds)) > 1}{/if}
-            <td class='scb_s_facs_samples_table_td {if $r.display_sample}{else}scb_s_experiment_setup_table_border{/if}' style="{if $r.kind == 'existing' and length(keys($kinds)) > 1}width:230px;{else}width:224px;{/if}">{if $r.display_sample}
-                {$r.display_text}
+            </td>
+            <td class='scb_s_facs_samples_table_td {if $r.display_sample}{else}scb_s_experiment_setup_table_border{/if}'>
+                {if $r.display_sample}
+                    {$r.display_text}
                 {/if}
             </td>
-            <td class='scb_s_facs_samples_table_td {if $r.display_sample}{else}scb_s_experiment_setup_table_border{/if} scb_s_facs_cell_treatment' style="{if $r.kind == 'existing' and length(keys($kinds)) > 1}width:171px;{else}width:136px;{/if}">
-            {if $r.is_sample_enabled}			 
-            	<input type="radio"  checked>
-            	    <span class='scb_s_facs_cell_treatment_text'>Fixed</span>
-            	</input>
-            {else}
-                <input type="radio" disabled>
-                    <span class='scb_s_facs_cell_treatment_text scb_s_microscopy_dim_conditions'>Fixed</span>
-                </input>
-            {/if}
+            <td class='scb_s_facs_samples_table_td {if $r.display_sample}{else}scb_s_experiment_setup_table_border {/if}
```

Comment:
It seems weird to control css classes with inline if statements, but it is clear enough to understand.  I almost wonder if something like:
```html
<script type="text/javascript">
{if not $r.display_sample}
$(.....).addClass('scb_s_experiment_setup_table_border')
</script>
```
to do it one place would save a lot of code.  Also, is there not a `{if not ...` in this template or similar?  The `else` seems unnecessary for all of these.